corehq/apps/reports/commtrack/data_sources.py

```diff
@@ -461,6 +461,7 @@ def get_data(self):
             )
 
             form_xmlnses = [form['xmlns'] for form in self.all_relevant_forms.values()]
+            form_xmlnses.append(COMMTRACK_REPORT_XMLNS)
```

Comment:
This somewhat changes the meaning of the report.  If someone DOES want to use the filter to show reporting rates from a specific form, AND the project has SMS stock updates, then those would obscure the data.  In my opinion, that's enough of an edge case that this solution wins the tradeoff, considering that currently there's no way to see consider data submitted by SMS.